models/DataObject/ClassDefinition.php

```diff
@@ -597,15 +597,9 @@ protected function getInfoDocBlock()
 
         $cd .= '/** ';
         $cd .= "\n";
-        $cd .= '* Generated at: '.date('c')."\n";
         $cd .= '* Inheritance: '.($this->getAllowInherit() ? 'yes' : 'no')."\n";
         $cd .= '* Variants: '.($this->getAllowVariants() ? 'yes' : 'no')."\n";
 
-        $user = Model\User::getById($this->getUserModification());
-        if ($user) {
-            $cd .= '* Changed by: '.$user->getName().' ('.$user->getId().')'."\n";
-        }
-
         if (isset($_SERVER['REMOTE_ADDR'])) {
             $cd .= '* IP: '.$_SERVER['REMOTE_ADDR']."\n";
```

Comment:
Another look at the PR, I think we also need to remove the IP. Agreed?

Reply:
Yep

Reply:
Made the changes btw